theano/gpuarray/ROIPoolGPUFwd.c

```diff
@@ -1,9 +1,9 @@
 #section kernels
 
-#kernel ROIPoolGPUFwd_kernel : ssize, ssize, *, ssize, ssize, ssize, ssize, ssize, ssize, *, *, * :
+#kernel ROIPoolGPUFwd_kernel : int32, int32, *, float32, int32, int32, int32, int32, int32, *, *, * :
```

Comment:
I don't think so, but @abergeron is/should work on the doc "shortly". So I think it is better to not duplicate work/do incompatible changes. Also this PR fix something to make this usable. Otherwise it wasn't usable.